src/test/java/com/google/cloud/tools/maven/it/AbstractMojoIntegrationTest.java

```diff
@@ -104,4 +107,53 @@ private static String getUrlContentWithRetries(String url, long timeoutMs, long
 
     return content;
   }
+
+  protected Verifier createStandardVerifier(String testName)
```

Comment:
I understand this is programming style thing, but would prefer these utilities be in their own classes and called into from the actual tests themselves.